### ISSUE TITLE
Oddness in layout just before mobile "navigation" menu takes effect

### ISSUE THREAD

**Issue by cmacdonell** [swcarpentry/website-archive#63]
![image](https://cloud.githubusercontent.com/assets/1037151/11453002/8f2628d4-95b7-11e5-958a-c9cb5dfc6826.png)

**Comment by cmacdonell** [comment]
My hunch is that line 76 in _sass/foundation-components/_top-bar.scss is the reason.  I think we need the mobile layout to kick-in sooner as width or height shrink

Here are the lines 75-77 of that file:
// Using rem-calc for the below breakpoint causes issues with top bar
$topbar-breakpoint: #{lower-bound($medium-range)} !default; // Change to 9999px for always mobile layout
$topbar-media-query: $medium-up !default;

**Comment by gvwilson** [comment]
@rgaiacs compared the site to http://datacarpentry.org, which uses the same template, and reports:

```
Data Carpentry has

    <div class="medium-4 columns"> <h2 class="font-size-h3 t10">Host a Workshop</h2> <a href="/workshops-host/"> <img src="http://datacarpentry.github.io/images//DC_room3.jpg" width="302" height="182" alt=""></a>

And we have

    <div class="small-4 columns"> <a href="/workshops/"><img src="http://dev.software-carpentry.org/files/workshops/2012-12-uta.png" alt="University of Texas 2012" /></a> <a href="/workshops/"><h4>Our Workshops ›</h4></a>
```

**Comment by todrobbins** [comment]
Yeah, it looks like the amount of nav items in the the nav bar is reflowing before the breakpoint. Data Carpentry doesn't do this because it has fewer items in the nav bar.

**Comment by gvwilson** [comment]
So is the only fix for me to reduce the number of items in the nav bar?  
Or is there some change to the CSS that'll make this more graceful?

**Comment by todrobbins** [comment]
@gvwilson like @cmacdonell recommmended, I think we just change the breakpoint to happen sooner before the stacking of items. I can work on this as well.

**Comment by gvwilson** [comment]
@cmacdonell can you please check the current http://dev.software-carpentry.org and let us know if this has been fixed?

**Comment by cmacdonell** [comment]
It appears to not be fixed when shrinking the width

![image](https://cloud.githubusercontent.com/assets/1037151/11568988/793bf776-99ac-11e5-930c-2428f1b8acf0.png)

**Comment by andreabedini** [comment]
I'm looking into this. I'm trying to make the break sooner by adjusting `$medium-range` and `$large-range`, in `_03_settings_mixins_media_queries.scss`. Currently medium breaks in at 64em × 16px = 1024px, while the nav items crash into each other at 1050px.

Question, why are including bootstrap too?

**Comment by todrobbins** [comment]
See #89 as another approach to resolving this

**Comment by gvwilson** [comment]
Anyone willing to take this on?

**Comment by andreabedini** [comment]
@gvwilson is this still an issue? My guess is that this was fixed by e248b5ea0eb03cc6db92dbec4ba3101df4768bc8. Opening www.software-carpentry.org I can't reproduce the [issue above](https://github.com/swcarpentry/website/issues/63#issuecomment-161731447).

**Comment by gvwilson** [comment]
If it isn't showing up for you, I'll close until it reappears - thanks for checking.
